messaging/client.go

```diff
@@ -341,9 +341,10 @@ func (c *Client) do(method, path string, values url.Values, contentType string,
 		// If a temporary redirect occurs then update the leader and retry.
 		// If a non-200 status is returned then an error occurred.
 		if resp.StatusCode == http.StatusTemporaryRedirect {
+			_ = resp.Body.Close()
```

Comment:
Nope. It's a habit I do from using `errcheck` on other projects. Removed in 00ce4a5.